frontend/src/components/PlayersBoard.js

```diff
@@ -0,0 +1,156 @@
+import styled from 'styled-components/macro';
+import ButtonTab from './ButtonTab';
+import PlayerTile from './PlayerTile';
+import ButtonBig from './ButtonBig';
+import { useState } from 'react';
+import getRandomColorCode from '../services/getRandomColorCode';
+
+export default function PlayersBoard({
+    existingPlayers,
+    setExistingPlayers,
+    selectPlayer,
+}) {
+    const [players, setPlayers] = useState([]);
+    const [playerToAdd, setPlayerToAdd] = useState('');
+    const [error, setError] = useState(false);
+    const [noPlayersSelected, setNoPlayersSelected] = useState(false);
+
+    const hasPlayerToAdd = playerToAdd.length > 0;
+
+    const handleChange = (event) => {
+        setError(false);
+        setPlayerToAdd(event.target.value);
+    };
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        if (!hasPlayerToAdd) {
+            return;
+        }
+        const playerToAddToExistingPlayers = {
+            name: playerToAdd,
+            color: getRandomColorCode(),
+        };
+        if (existingPlayers.length > 0) {
+            if (
+                !existingPlayers.find(
+                    (existingPlayer) => existingPlayer.name === playerToAdd
+                )
+            ) {
+                setExistingPlayers([
+                    ...existingPlayers,
+                    playerToAddToExistingPlayers,
+                ]);
+                setPlayerToAdd('');
+            } else {
+                setError(true);
+            }
+        }
+    };
+
+    const onChange = (selectedPlayer, event) => {
+        setNoPlayersSelected(false);
+        if (event.target.checked) {
+            if (
+                !players.includes(
+                    (player) => player.name === selectedPlayer.name
+                )
+            ) {
+                setPlayers([...players, selectedPlayer]);
+            }
+        } else {
+            setPlayers(
+                players.filter((player) => player.name !== selectedPlayer.name)
+            );
+        }
+    };
+
+    const handleClick = (event) => {
+        if (players.length > 0) {
+            selectPlayer(players);
+        } else {
+            setNoPlayersSelected(true);
+        }
+    };
+
+    return (
+        <>
+            <Wrapper>
+                <Form onSubmit={handleSubmit}>
+                    <input
+                        type="text"
+                        value={playerToAdd}
+                        placeholder="NEW PLAYER"
+                        onChange={(event) => handleChange(event)}
+                    />
+                    <ButtonBig type="submit" disabled={!hasPlayerToAdd}>
+                        +
+                    </ButtonBig>
+                </Form>
+                {error && <p>Player already exists!</p>}
+                {noPlayersSelected && <p>No players selected!</p>}
+                <div className="listbox">
```

Comment:
I think you can get rid of this div and do `overflow-y: scroll` on the list?

Reply:
you are right ^^